build.js

```diff
@@ -357,9 +357,6 @@ export async function bundleCombinedWorkers(options) {
   esBuildConfig.entryPoints = workers;
   esBuildConfig.outdir = options.path;
   esBuildConfig.minify = options.minify;
-  esBuildConfig.banner = {
```

Comment:
We still want our workers (`packages/engine/Source/Workers/**`) to include the copyright banner, correct?